src/Home.tsx

```diff
@@ -38,14 +42,21 @@ export const Home: React.FC = () => {
   >();
   const { colorMode, toggleColorMode } = useColorMode();
   const w3Client = useWeb3ApiClient();
+  const { active, activate, account, deactivate } = useWeb3React();
 
   const logMsgHandler = async (): Promise<any> => {
     setSubmitted(true);
   };
 
+  // Need help setting account from connected wallet
+  const setConnectHandler = async (): Promise<any> => {
+    setAccountAddress(account);
```

Comment:
You can try and use this ```
const [accountNum] = useState<string>(
        account ? `${account.substring(0, 6)}...${account.substring(account.length - 4)}` : '',
    );
``` It lets the user see the last four of their address

Reply:
You also need to use `setSubmitted(true)` so that user don't need to submit manually after connecting the wallet.